pkg/apis/resource/validation/validation.go

```diff
@@ -226,72 +126,12 @@ func ValidateClaimStatusUpdate(resourceClaim, oldClaim *resource.ResourceClaim)
 func validateAllocationResult(allocation *resource.AllocationResult, fldPath *field.Path) field.ErrorList {
 	var allErrs field.ErrorList
 	if allocation != nil {
-		if len(allocation.ResourceHandles) > 0 {
-			allErrs = append(allErrs, validateResourceHandles(allocation.ResourceHandles, resource.AllocationResultResourceHandlesMaxSize, fldPath.Child("resourceHandles"))...)
+		// if len(allocation.ResourceHandles) > 0 {
+		// 	allErrs = append(allErrs, validateResourceHandles(allocation.ResourceHandles, resource.AllocationResultResourceHandlesMaxSize, fldPath.Child("resourceHandles"))...)
+		// }
```

Comment:
Why commented out instead of removed?

Reply:
Because validation.go needs more work. At the moment, it only compiles because I commented out everything that hasn't been adapted yet.